Simplified/NYPLOPDSEntry.m

```diff
@@ -135,9 +135,9 @@ - (instancetype)initWithXML:(NYPLXML *const)entryXML
   self.providerName = [entryXML firstChildWithName:@"distribution"].attributes[@"bibframe:ProviderName"];
   
   {
-    NSString *const dateString = [entryXML firstChildWithName:@"published"].value;
+    NSString *const dateString = [entryXML firstChildWithName:@"issued"].value;
     if(dateString) {
-      self.published = [NSDate dateWithRFC3339String:dateString];
+      self.published = [NSDate dateWithISO8601DateString:dateString];
```

Comment:
Test this code for me by changing this line to 
`self.published = nil` and make sure the UI still looks okay.

Reply:
When there is a valid published date, it will look like this:
![simulator screen shot - iphone 8 plus - 2018-11-12 at 14 30 47](https://user-images.githubusercontent.com/1022275/48376960-b7998600-e689-11e8-9ccd-3c882445936c.png)

To perform that nil test, when the published date is nil, the screen for the same book looks like this:
![nodate](https://user-images.githubusercontent.com/1022275/48376968-bb2d0d00-e689-11e8-9464-5de769d1c2cb.png)

Which is similar to what it would look like for a book that typically has no published date, such as this one from the publisher Project Guttenberg (no Published line gets displayed at all):
![simulator screen shot - iphone 8 plus - 2018-11-12 at 14 31 33](https://user-images.githubusercontent.com/1022275/48376981-c2ecb180-e689-11e8-8a2e-7d82d2c122c2.png)

@gioneill Let me know if you have any additional comments/questions. Thanks Greg!